### PR TITLE
chore: post v7.2.0 release

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

[Compatibility tests for v7.2.x passed before release](https://github.com/cosmos/ibc-go/actions/runs/5349173084).

Compatbility tests updated with @charleenfei and @DimitrisJim script! 🤖 

I will consider v7.0.1, v7.1.0, release-v7.0.x and release-v7.1.x unsupported for now. If it turns out that the fix in 7.2.0 can be back ported, we can support those lines again.

closes: #XXXX

### Commit Message / Changelog Entry

```bash
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [x] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: .github/compatibility-test-matrices/main/client-chain-a.json [hunk 1]
```diff
@@ -32,4 +31,4 @@
   "chain-image": [
     "ghcr.io/cosmos/ibc-go-simd"
   ]
-}
+}
```
Comment (DimitrisJim) [inline .github/compatibility-test-matrices/main/client-chain-a.json:34]: ideally this noise should disappear in the future, the script currently trims the last empty line!

File: .github/compatibility-test-matrices/release-v7.2.x/client-chain-b.json [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
     "v6.2.0",
     "v6.1.1",
+    "v5.3.1",
     "v5.2.1",
     "v4.4.2",
     "v4.3.1",
     "v4.2.2",
     "v4.1.3",
-    "v5.3.1"
+    "release-v7.2.x"
```
Comment (DimitrisJim) [inline .github/compatibility-test-matrices/release-v7.2.x/client-chain-b.json:12]: aha, we might also want this to be first

Comment (crodriguezvega) [reply .github/compatibility-test-matrices/release-v7.2.x/client-chain-b.json:12]: Either top or bottom works for me. :)

**Review by damiannolan — APPROVED** [review]

**Review by chatton — APPROVED** [review]
LGTM

**Pull request merged**